lib/webinars.rb

```diff
@@ -36,41 +37,62 @@ def find(webinar_id)
     end
 
     def add_panelist(webinar:, user:)
-      response = zoom_client.post("webinars/#{webinar.zoom_id}/panelists",
-        panelists: [{
-          email: user.email,
-          name: user.name.blank? ? user.username : user.name
-        }]
-      )
+      response =
+        zoom_client.post(
+          "webinars/#{webinar.zoom_id}/panelists",
+          panelists: [
+            {
+              email: user.email,
+              name: user.name.blank? ? user.username : user.name
+            }
+          ]
+        )
       return false if response.status != 201
 
       WebinarUser.where(user: user, webinar: webinar).destroy_all
       WebinarUser.create!(user: user, webinar: webinar, type: :panelist)
     end
 
     def remove_panelist(webinar:, user:)
-      panelists = zoom_client.panelists(webinar.zoom_id, true)[:body][:panelists]
-      matching_panelist = panelists.detect do |panelist|
-        panelist[:email] == user.email
-      end
+      panelists =
+        zoom_client.panelists(webinar.zoom_id, true)[:body][:panelists]
+      matching_panelist =
+        panelists.detect { |panelist| panelist[:email] == user.email }
       return false unless matching_panelist
 
-      response = zoom_client.delete("webinars/#{webinar.zoom_id}/panelists/#{matching_panelist[:id]}")
+      response =
+        zoom_client.delete(
+          "webinars/#{webinar.zoom_id}/panelists/#{matching_panelist[:id]}"
+        )
       return false if response.status != 204
 
       WebinarUser.where(user: user, webinar: webinar).destroy_all
     end
 
     def signature(webinar_id)
-      return false unless SiteSetting.zoom_api_key && SiteSetting.zoom_api_secret
+      unless SiteSetting.zoom_sdk_key && SiteSetting.zoom_api_secret
```

Comment:
I know it was `unless` before.. but can you convert this to an `if`? It's super hard to understand `unless`s with `&&` or `||` involved.